Mixpanel/MixpanelInstance.swift

```diff
@@ -1135,7 +1140,7 @@ extension MixpanelInstance: InAppNotificationsDelegate {
     // MARK: - Decide
     func checkDecide(forceFetch: Bool = false, completion: @escaping ((_ response: DecideResponse?) -> Void)) {
         self.trackingQueue.async {
-            self.networkQueue.async {
+            self.networkQueue.sync {
```

Comment:
does this mean decide blocks the tracking queue until the network request completes?